Applications/Telemetry/telemetry.py

```diff
@@ -24,10 +25,11 @@
 from flask import request
 from flask_cors import CORS
 
-# Can we clean this up?
-sys.path.append(os.path.join(os.path.dirname(__file__), "../../Faraday_Proxy_Tools/"))  #Append path to common tutorial FaradayIO module
-from FaradayIO import faradaybasicproxyio
-from FaradayIO import telemetryparser
+# Add Faraday library to the Python path.
+sys.path.append(os.path.join(os.path.dirname(__file__), '../..'))
```

Comment:
I get an import error when trying to run telemetry.py

![image](https://cloud.githubusercontent.com/assets/339144/25066965/0c5d8a80-21ea-11e7-86cf-46cfcae6c8f6.png)